packages/server/lib/modes/run.js

```diff
@@ -1154,11 +1171,13 @@ module.exports = {
         }
       }
 
-      // always close the browser now as opposed to letting
-      // it exit naturally with the parent process due to
-      // electron bug in windows
-      debug('attempting to close the browser')
-      await openProject.closeBrowser()
+      if (!runAllSpecsInSameBrowserSession) {
```

Comment:
Do we even want to keep this flag? It looks like there is no way to access it programatically - should we just make this the default?

Reply:
I think that's what's happening. We're setting this as the default for CT, but because this run code is shared, you need to plumb the options through

Reply:
Probably. Is it the right PR to test this change?

Reply:
In e2e mode, this flag is false by default. We need to keep the behavior as is for e2e's sake for now.

They can choose to opt-in to this faster tooling when they wish to.

For now, I prefer to keep it a separate option.